app/src/main/java/com/practicum/playlistmaker/ui/root/search/SearchFragment.kt

```diff
@@ -49,6 +51,12 @@ class SearchFragment : Fragment(), SearchScreenView {
         viewModel.addAllToSaveHistory(historyList)
     }
 
+    override fun onResume() {
+        super.onResume()
+        if (savedText.isNullOrEmpty())
```

Comment:
🍏 Переменная savedText не нуллабельна
```suggestion
        if (savedText.isEmpty())
```